src/components/chatInput/hoc/withUserSearch.js

```diff
@@ -0,0 +1,34 @@
+import { withStateHandlers, compose, withProps } from 'recompose';
+import searchUsers from 'shared/graphql/queries/search/searchUsers';
+import { flatMap, map } from 'lodash';
```

Comment:
nit: let's get rid of this lodash usage since we don't use it anywhere on the client and don't really need it here either:

```JS
data.search ? data.search.searchResultsConnection.edges.map(({ node }) => node) : [];
```